WTWSMS/common/landed_titles/WtWSMS_pagan_religious_heads.txt

```diff
@@ -632,7 +632,7 @@ d_jewish = {
 	dynasty_title_names = no 	# Will not be named "Seljuk", etc.
 }
 
-d_melchisedechianism = {
+k_melchisedechianism = {
```

Comment:
Burden of proof lies on you here. Show me _why_, I'm open to reconsidering my stance.